pipeline/scripts/analysis_pipeline.jl

```diff
@@ -0,0 +1,85 @@
+# Analogy: library(targets) in R
+using DrWatson
+
+# Activate the project environment
+# Analogy: source(functions.R) in targets
+@quickactivate "Analysis pipeline"
+
+# Include the AnalysisPipeline module
+include(srcdir("AnalysisPipeline.jl"));
+# Include some plotting functionality that is useful but not necessary for analysis
+# pipeline
+include(plotsdir("plot_functions.jl"));
+
+@info("""
+      Running the analysis pipeline.
+      ---------------------------------------------
+      Currently active project is: $(projectname())
+      Path of active project: $(projectdir())
+      """)
+
+using .AnalysisPipeline
```

Comment:
i.e under the pipeline is generic this would become `EpiAwarePipeline` and we would also have `RtwithoutRenewalAnalysis` with the specific scenario grid and plots etc?

Reply:
looking at below though I like it as a "default" so perhaps those would always stay in the pipeline module?

Reply:
also if it didn't it would be very full as its mostly defaults :)